go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/improbable-eng/grpc-web v0.12.0
 	github.com/logrusorgru/aurora v2.0.3+incompatible
 	github.com/onflow/cadence v0.27.0
-	github.com/onflow/flow-go v0.26.14-test-synchronization.0.20221006164531-ac5e7c1492bc
+	github.com/onflow/flow-go v0.26.14-test-synchronization.0.20221006192037-6066b77a0aae
```

Comment:
I think this was a "in-between" PR of the PR stack, so this dependency update change now actually downgrades. It probably has to be reverted